Sources/Graphics/image/BufferedImage.swift

```diff
@@ -2,7 +2,7 @@
  * A mutable image that provides direct access to its pixels.
  */
 public protocol BufferedImage: Image {
-    init(width: Int, height: Int) throws
+    init(width: Int, height: Int, format: PixelFormat) throws
```

Comment:
Could we add `init(width:height:)` as a protocol extension here  too (for convenience and to avoid breaking consumers)?